ee/clickhouse/models/test/test_cohort.py

```diff
@@ -763,7 +765,7 @@ def test_cohortpeople_with_misdirecting_cyclic_cohort_filter(self):
         # cohort1 depends on cohort2 which depends on cohort3 which depends on cohort4 which depends on cohort2
         # and cohort5 depends on cohort1
 
-        with self.assertRaises(ValueError):
+        with self.assertRaises(RecursionError):
```

Comment:
eh, don't need 2 of these tests now that it should be impossible to get into this state. Will remove.